docs/TutorEnvironmentManager.rst

```diff
@@ -0,0 +1,108 @@
+Tutor Environment Manager
+#########################
+
+Tutor environment manager allows you to create one project and set one version for it.
+
+You will be able to continue using the `tutor version manager`_ commands in your project:
+
+- tvm list
+- tvm plugins list
+
+
+Creating a project
+------------------
+
+To create a new project you should call ``tvm project init``, this configures a new tvm project in the current path and set your-project as ``TUTOR_ROOT`` and ``TUTOR_PLUGINS_ROOT``.  
```

Comment:
Maybe we need to specify that TUTOR_ROOT and TUTOR_PLUGINS_ROOT must be absolute paths

Reply:
I agree but adding in version manager, configuring tvm variables section, where describe how to change it and because project configures it automatically. 

Let me know your opinion.